torch_ttnn/passes/lowering/add_data_move_pass.py

```diff
@@ -23,12 +23,129 @@ def is_function_call(node) -> bool:
     return node.op == "call_function"
 
 
+TTNN_POINTWISE_UNARY_OPS = [
+    ttnn.abs,
+    ttnn.acos,
+    ttnn.acosh,
+    ttnn.asin,
+    ttnn.asinh,
+    ttnn.atan,
+    ttnn.atan2,  # binary
+    ttnn.atanh,
+    #  ttnn.clone,  in target_wrappers
+    ttnn.cos,
+    ttnn.cosh,
+    ttnn.erf,
+    ttnn.exp,
+    ttnn.expm1,
+    ttnn.gelu,
+    ttnn.hardtanh,
+    ttnn.isinf,
+    ttnn.isnan,
+    ttnn.leaky_relu,
+    ttnn.log,
+    ttnn.log10,
+    ttnn.log1p,
+    ttnn.log2,
+    ttnn.logical_not,
+    ttnn.neg,
+    ttnn.reciprocal,
+    ttnn.relu,
+    ttnn.rsqrt,
+    ttnn.sigmoid,
+    ttnn.softmax,
+    ttnn.sign,
+    ttnn.sin,
+    ttnn.sinh,
+    ttnn.silu,
+    ttnn.sqrt,
+    ttnn.tan,
+    ttnn.tanh,
+]
+
+
+TTNN_POINTWISE_BINARY_OPS = [
+    ttnn.add,
+    ttnn.eqz,
+    ttnn.gez,
+    #ttnn.gte,  # ttnn has no gte, remote the comment in the future when it has
+    ttnn.gtz,
+    ttnn.isclose,
+    ttnn.ldexp,
+    ttnn.lez,
+    ttnn.logaddexp,
+    ttnn.logaddexp2,
+    #ttnn.lte,  # ttnn has no lte, remote the comment in the future when it has
+    ttnn.ltz,
+    ttnn.nextafter,
+    ttnn.nez,
+    ttnn.polyval,
+    ttnn.squared_difference,
+    ttnn.eq,
+    ttnn.gt,
+    ttnn.logical_and,
+    ttnn.logical_or,
+    ttnn.logical_xor,
+    ttnn.lt,
+    ttnn.maximum,
+    ttnn.minimum,
+    ttnn.mul,
+    ttnn.ne,
+    ttnn.pow,
+    ttnn.sub,
+    ttnn.xlogy,
+    #ttnn.add_and_apply_activation,  # ttnn has no add_and_apply_activation, remote the comment in the future when it has
+    #ttnn.add_and_apply_activation_,  # ttnn has no add_and_apply_activation, remote the comment in the future when it has
```

Comment:
What is add and apply activation? is it add with bias?

Reply:
I have no idea either. The best match I can find is tenstorrent/tt-metal#8116